### PR TITLE
added eli

### PR DESCRIPTION
**Pull request by elischutze**
@tanyapowell @krissy

I put my **beautiful face** on the slides
and added a quick "bio"

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -119,12 +119,14 @@
         </section>
 
         <section>
-          <img src="http://placehold.it/250x250">
```
Comment (tanyapowell) [inline index.html:122]: could you add `id="bio-pic"` to your `` tag pretty please :dancer:

Comment (elischutze) [inline index.html:122]: I tried but it squishes my face

**Comment by tanyapowell** [comment]
:ship: it :+1: :dancers:

**Comment by elischutze** [comment]
📝✅

**Pull request merged**